cmd/livepeer/livepeer.go

```diff
@@ -724,42 +711,10 @@ func main() {
 		}()
 	}
 
-	if *s3bucket != "" && *s3creds == "" || *s3bucket == "" && *s3creds != "" {
-		glog.Error("Should specify both s3bucket and s3creds")
-		return
-	}
-	if *gsBucket != "" && *gsKey == "" || *gsBucket == "" && *gsKey != "" {
-		glog.Error("Should specify both gsbucket and gskey")
-		return
-	}
-
-	// XXX get s3 credentials from local env vars?
-	if *s3bucket != "" && *s3creds != "" {
-		br := strings.Split(*s3bucket, "/")
-		cr := strings.Split(*s3creds, "/")
-		u := url.URL{
-			Scheme: "s3",
-			Host:   br[0],
-			Path:   fmt.Sprintf("/%s", br[1]),
-			User:   url.UserPassword(cr[0], cr[1]),
-		}
-		glog.Warningf("-s3bucket and -s3creds are deprecated. Instead, you can use -objectStore %s", u.String())
-		ustr := u.String()
-		objectstore = &ustr
-	}
-
-	if *gsBucket != "" && *gsKey != "" {
-		u := url.URL{
-			Scheme:   "gs",
-			Host:     *gsBucket,
-			RawQuery: fmt.Sprintf("keyfile=%s", *gsKey),
-		}
-		glog.Warningf("-gsbucket and -gskey are deprecated. Instead, you can use -objectStore %s", u.String())
-		ustr := u.String()
-		objectstore = &ustr
-	}
-
 	if *objectstore != "" {
+		if driversTesting := os.Getenv("OBJECT_STORE_TESTING"); driversTesting == "true" {
```

Comment:
Would it be possible get the test_args.sh test involving `-objectStore` to pass without this for example by passing in stub S3 object store values? Or does that break something?

Reply:
I had tried `gs://foo` , but got `unexpected end of JSON output` 

Then I tried `memory://foo` but this required `drivers.Testing = true`

Reply:
Did you try the suggestion in [this comment](https://github.com/livepeer/go-livepeer/pull/1838#discussion_r611809224)?

Reply:
Ah yes that seems to work c9613c1